lib/datadome/inquirer.rb

```diff
@@ -26,6 +27,11 @@ def enriching
 
       headers.merge!(added_headers)
 
+      if @monitoring_mode
```

Comment:
WDYT of adding a dedicated variable like `expose_headers`?

Reply:
good idea